### PR TITLE
Update webpack config to build Sass correctly.

### PR DESCRIPTION
**Pull request by douglashall**

### CODE REVIEW

File: config/webpack.prod.config.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const Merge = require('webpack-merge');
 const commonConfig = require('./webpack.common.config.js');
 const path = require('path');
 const webpack = require('webpack');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
```
Comment (georgebabey) [inline config/webpack.prod.config.js]: You can remove ExtractTextPlugin now.

File: package.json [hunk 2]
```diff
@@ -69,6 +69,7 @@
     "husky": "^0.14.3",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^22.4.0",
+    "mini-css-extract-plugin": "^0.4.0",
```
Comment (georgebabey) [inline package.json]: Can remove `extract-text-webpack-plugin` from here too (up 8 lines above this one)

**Comment by douglashall** [comment]
@robrap @reillz10 Do you want me to merge this into your branch?

**Comment by robrap** [comment]
@reillz10: Do you want this merged to the footer branch, or to master?

**Comment by douglashall** [comment]
Closing this in favor of https://github.com/edx/gradebook/pull/51

**Pull request closed without merging**